### DISCUSSION TITLE
手表无法获取天气数据

### DISCUSSION THREAD

**Discussion by jaykwok** [NSRingo/.github#6] [Q&A 问答] [answered]
版本信息：IOS18，watchOS11，使用了最新的weatherkit，手机显示正常，手表无法获取天气信息。
尝试看网络记录，发现是weatherkit.apple.com被丢包了，经查询前几个版本的日志，确实是有这一条。请问手表的话，这边该怎么设置呢，尝试看首页的watch wiki，但是跳转到404页面了，估计是被删了？谢谢

**Comment by jaykwok** [comment, marked as answer]
已解决，quantumultx里面。配置证书——下载证书——safari打开后选择apple watch，安装后即可

**Comment by champeil** [comment]
请问stash中如何解决呢？
